pkg/oc/bootstrap/docker/fileutils_windows.go

```diff
@@ -0,0 +1,171 @@
+// +build windows
```

Comment:
Is this actually a fully generic implementation that works for linux and windows?  If so, I definitely prefer it.